storybook/stories/components/card/filter.js

```diff
@@ -69,7 +74,9 @@ export default function() {
                 <ListItem index={4} />
 
                 <Grid>
-                  <p>{loremIpsum.substr(0, 100)}</p>
+                  <p>
+                    {loremIpsum.substr(0, 100)}
+                  </p>
```

Comment:
Yeah I'm pretty sure my home laptop has a different version of the prettier plugin. Although I thought we had a pre commit hook that ran prettier? Which I would expect to be the same version for the project?